### PR TITLE
fix(curriculum): add explanation for child combinator

### LINKED ISSUE #51235: CSS child combinator could use better explanation
### Describe the Issue

I believe that Accessibility Quiz step 13 is the first time we introduce the CSS child combinator. (I've looked through the finished CSS for all previous courses and have not found it.) The instructions and hint for step 13 merely say to use it but they don't explain what it does or why it is needed. And really, since the only child in the `nav` is the `ul`, and there are no embedded `ul`s in the list items, then it is not obvious why it is needed in the first place. We also require the child combinator in later steps and I am not finding any explanation for it in those steps either.

### Affected Page

https://www.freecodecamp.org/learn/2022/responsive-web-design/learn-accessibility-by-building-a-quiz/step-13

### Your code

The current instructions for step 13:

> Use the > selector to target the unordered list elements within the nav elements, and use Flexbox to evenly space the children.

The hint if you don't use the child combinator:

> You should use the nav > ul selector.

### Expected behavior

Add additional information to the instructions:

- Give the name of the `>` selector (child combinator)
- Describe what it is used for
- Explain why it is being used in this step

### Screenshots

_No response_

### System

N/A

### Additional context

I don't have any suggested text at the moment, I just wanted to get this issue recorded so I wouldn't forget about it. And to be honest, I'm not sure what the rationale is for requiring the child combinator here. Perhaps we are just using it merely for the sake of introducing a new feature in CSS?

### PR DESCRIPTION
**Pull request by evankart**
Checklist:

- [x] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [x] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request).
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Closes #51235

### CODE REVIEW

**Review by gikf — APPROVED** [review]

File: curriculum/challenges/english/14-responsive-web-design-22/learn-accessibility-by-building-a-quiz/614090d5a22b6f0a5a6b464c.md [hunk 1]
```diff
@@ -9,7 +9,11 @@ dashedName: step-13
 
 # --description--
 
-Use the `>` selector to target the unordered list elements within the `nav` elements, and use _Flexbox_ to evenly space the children.
+The child combinator selector `>` is used between selectors to target only elements that match the second selector and are a direct child of the first selector.
+
+This can be helpful when you have deeply nested elements and want to control the scope of your styling.
+
+Use the `>` selector to target the unordered list elements within the `nav` elements, and use Flexbox to evenly space the children.
```
Comment (jdwilkin4) [inline curriculum/challenges/english/14-responsive-web-design-22/learn-accessibility-by-building-a-quiz/614090d5a22b6f0a5a6b464c.md:16]: It looks like the original here had the word Flexbox italicized so we should probably keep that part that same.

```suggestion
Use the `>` selector to target the unordered list elements within the `nav` elements, and use _Flexbox_ to evenly space the children.
```

**Review by jdwilkin4 — CHANGES_REQUESTED** [review]
Looks good. Left one small comment

File: curriculum/challenges/english/14-responsive-web-design-22/learn-accessibility-by-building-a-quiz/614090d5a22b6f0a5a6b464c.md (hunk 1 above)
Comment (evankart) [reply curriculum/challenges/english/14-responsive-web-design-22/learn-accessibility-by-building-a-quiz/614090d5a22b6f0a5a6b464c.md]: Agreed - thanks @jdwilkin4!

**Review by jdwilkin4 — APPROVED** [review]
Congrats on your first PR being approved into the freeCodeCamp curriculum 🎉

**Pull request merged**